recipes/phe/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "phe" %}
+{% set version = "1.4.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: b5785c8e824eb7b02ebbe9b1e2080982d48a10db7b827a6771e04e22da99f473
+
+build:
+  number: 0
+  entry_points:
+    - pheutil = phe.command_line:cli
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+  run_constrained:
+    - gmpy2
+    - click
+    - pycrypto
+
+test:
+  imports:
+    - phe
+    - phe.tests
+  commands:
+    - pheutil --help
+  requires:
+    - click
+    - gmpy2
+    - numpy
+
+about:
+  home: "https://github.com/data61/python-paillier"
+  license: GPL-3.0-only
```

license is wrong since parts are apache

---

which parts?

---

Thanks for the feedback Matthew.
I was under the impression that it is okay to include Apache licensed software in GPL3 projects, as [stated by the Apache software foundation](http://www.apache.org/licenses/GPL-compatibility.html).

---

It is, but to use the software you have to comply with both licences:
```suggestion
  license: GPL-3.0-only AND Apache-2.0
```

---

Oh, I see. Thanks, Chris.